### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by ghadimq** [fight-think/features-extraction#1]
hello, is the (mc) file mean it is a malicious certificate and the (pc) file mean it is benign?

**Comment by fight-think** [comment]
Yes, you are right. "m" in "mc.csv" means malicious. "p" in "pc.csv" means positive.
